dom/select-then-style_test.js

```diff
@@ -0,0 +1,49 @@
+
+export const tests = []
+
+tests.push(async ({ eq }) => {
+  // check the CSS stylesheet is linked in the head tag
+
+  await eq.$('head link', {
+    rel: 'stylesheet',
+    href: 'http://localhost:9898/select-and-style/select-and-style.css',
```

Comment:
should this be `select-then-style` instead of `select-and-style`?

Reply:
Adressed.